spring-boot/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-spring-boot</artifactId>
+    <parent>
+        <relativePath>../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+
+    <name>Infinispan Simple Tutorials: Spring Boot</name>
+
+    <properties>
+        <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
+        <infinispan.starters.version>1.0.0-SNAPSHOT</infinispan.starters.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-parent</artifactId>
+                <version>${spring-boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-embedded</artifactId>
+        </dependency>
```

Comment:
I am afraid that doesn't follow the principle of a Spring Boot starter then. If you look at `spring-boot-starter-web` we took the decision to bring Tomcat with it (because adding that starter should be all that's required to get you started for building a web application). If you don't like that, yep, you need to exclude the `spring-boot-starter-tomcat` and replace it with either the jetty or undertow one.

> Users who want to use embedded mode need to decide whether they want to use 

No they don't. They chose to use a starter and that starter is going to bring them the best practice and whatever you guys recommend (or want to push forward). Probably not the uber jar 😄 

> So as you can see, there are many different deployment scenarios here. 

Yes there is. That's fine. The starter should bring a default, something predictable with a minimum amount of required configuration. I would bring client/server by default (because it must be the most lightweight dependencies-wise), requiring the user to provide the location of the remote infinispan cluster. They could go embedded by adding the necessary code to do that (if that's not a single dependency, maybe a separate starter that would nicely play with this one?). Granted, they'd get both client/server and embedded but what is the actual problem with that?